Using UUID as service identifiers

eelcocramer opened the pull request:
Jira issue: WP-413

eelcocramer committed:
Added dependency node node-uuid

Jira issue: WP-413

eelcocramer committed:
Calculates a globbaly unique UUID as service identifier instead of having predicatable service identifiers.

WP-413

eelcocramer committed:
Added test case to test wheter it is possible to instantiate 2 services instances of the same type.

WP-413

habibvirji commented on package.json:
```diff
@@ -18,6 +18,7 @@
     "mdns": "0.x.x",
     "jsormdb": "0.1.x",
     "now": "0.8.1",
+	"node-uuid:": "1.3.x",
```
formatting is different

eelcocramer committed:
Fixed typo… :-(

Jira issue: WP-413

eelcocramer commented:
:-( fixed it.

eelcocramer committed:
Merge branch 'master' of git://github.com/webinos/Webinos-Platform into wp-413

eelcocramer committed:
Didn't realize that this was used in the browser as well and it needed to be independend of other node modules. This commit fixes that issue.
Also implement this for the id's generated in the browser context.

Jira issue: WP-413

eelcocramer commented:
Hold this pull request for a bit. It needs further analysis.
